src/components/Header/Header.module.css

```diff
@@ -0,0 +1,28 @@
+.header {
+  border-bottom: 1px solid var(--chakra-colors-gray-300);
+  height: 64px;
```

Comment:
Let's add new design tokens for 8/16/32/64px sizes. It seems we use this a lot.

Reply:
no.